package-lock.json

```diff
@@ -22966,7 +22967,7 @@
     },
     "@brave/leo": {
       "version": "git+ssh://git@github.com/brave/leo.git#dd81836705e7a0055f70aa15f1955ced61d624cc",
-      "integrity": "sha512-0r2Wh6iNILP4g2TBVxkhIfP7Y2x/7eCsvXvgLOYTy0NyT7MyfGhL581Mfez1zSrDSe5t97ROIPFNLdgz+JrwaA==",
+      "integrity": "sha512-/r+Je9x4A2SoHILT2tsF7wSLiY3SEzQVJYbyAL89DETdEdaum4f1ykmIeec6OcmnHNxZwxVOqC95sii8p6GHSg==",
```

Comment:
did this PR intend to update the leo packages? if not i think the changes to this file need to be removed.

Reply:
right. will do.